source/intro.md

```diff
@@ -631,6 +629,44 @@ ten_lang = arranged_lang.head(10)
 ten_lang
 ```
 
+## Adding and modifying columns using `assign`
+
```

Comment:
I moved this earlier (to avoid breaking the lead-up to viz) and made it clear that this was a bit of a diversion (and a teachable moment for forking exploration ;) )